packages/suite/src/hooks/wallet/coinmarket/form/useCoinmarketSellFormDefaultValues.ts

```diff
@@ -15,25 +15,31 @@ import {
     FORM_DEFAULT_PAYMENT_METHOD,
 } from 'src/constants/wallet/coinmarket/form';
 import { useCoinmarketBuildAccountGroups } from 'src/hooks/wallet/coinmarket/form/common/useCoinmarketBuildAccountGroups';
+import { useSelector } from 'src/hooks/suite';
 
 export const useCoinmarketSellFormDefaultValues = (
     account: Account,
     sellInfo: SellInfo | undefined,
 ): CoinmarketSellFormDefaultValuesProps => {
     const country = sellInfo?.sellList?.country;
     const cryptoGroups = useCoinmarketBuildAccountGroups('sell');
+    const prefilledFromCryptoId = useSelector(
+        state => state.wallet.coinmarket.prefilledFromCryptoId,
+    );
     const cryptoOptions = useMemo(
         () => cryptoGroups.flatMap(group => group.options),
         [cryptoGroups],
     );
     const defaultSendCryptoSelect = useMemo(
         () =>
+            (prefilledFromCryptoId &&
+                cryptoOptions.find(option => option.value === prefilledFromCryptoId)) ||
             cryptoOptions.find(
                 option =>
                     option.descriptor === account.descriptor &&
                     account.symbol === cryptoIdToNetworkSymbol(option.value),
             ),
-        [account.descriptor, account.symbol, cryptoOptions],
+        [account.descriptor, account.symbol, prefilledFromCryptoId, cryptoOptions],
```

Comment:
I think we need to set also `outputs.0.token` to work fine. You can check the _useCoinmarketFormActions_ function `onCryptoCurrencyChange` for more information.

This is related to the issue that I have found before I suppose.

Same issue for _useCoinmarketExchangeFormDefaultValues_

Reply:
Fixed in 25c58e011373a67052fb6a66bb21f2273c773efa